freqtrade/optimize/hyperopt.py

```diff
@@ -23,7 +23,8 @@
 from freqtrade.configuration import Arguments
 from freqtrade.data.history import load_data, get_timeframe
 from freqtrade.optimize.backtesting import Backtesting
-# Import IHyperOptLoss to allow users import from this file
+# Import IHyperOpt and IHyperOptLoss to allow unpickling classes from these modules
+from freqtrade.optimize.hyperopt_interface import IHyperOpt  # noqa: F4
```

Comment:
I'm not sure if this is really needed. Add it when I was fighting with the unpickling problems for ticker_interval class attribute...

Reply:
makes sense anyway ... just like HyperoptLoss - so users can import from here...